src/ProductManagement.java

```diff
@@ -0,0 +1,122 @@
+import com.sun.xml.internal.ws.policy.EffectiveAlternativeSelector;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class ProductManagement {
+    private ArrayList<Product> listProducts;
+
+    ProductManagement() {
+        listProducts = new ArrayList<Product>();
+    }
+
+    public void addProduct(Product product) {
+        listProducts.add(product);
+    }
+
+    public void sortByName() {
+        Collections.sort(listProducts);
+    }
+
+    public void sortFromLowestPrice() {
+        Collections.sort(listProducts, new Comparator<Product>() {
+            @Override
+            public int compare(Product product, Product t1) {
+                return (int) ((product.getPrice() - (t1.getPrice())) * 100);
+            }
+        });
+    }
+
+    public void sortFromHighestPrice() {
+        Collections.sort(listProducts, new Comparator<Product>() {
+            @Override
+            public int compare(Product product, Product t1) {
+                return -(int) ((product.getPrice() - (t1.getPrice()))* 100);
+            }
+        });
+//        display();
+    }
+
+    public void sortByProduction() {
+        Collections.sort(listProducts, new Comparator<Product>() {
+            @Override
+            public int compare(Product product, Product t1) {
+                return product.getProduction().compareTo(t1.getProduction());
+            }
+        });
+//        display();
+    }
+
+    public void editWithId(String editID, String newID, String newName, String newProduction, float newPrice, String newDescription) {
+        for (int i = 0; i < listProducts.size(); i++) {
+            Product product = listProducts.get(i);
+            if (product.getProductID().compareTo(editID) == 0) {
+                product.setName(newName);
+                product.setDescription(newDescription);
+                product.setPrice(newPrice);
+                product.setProductID(newID);
+                product.setProduction(newProduction);
+                System.out.println("Change completed");
+                return;
+            }
+        }
+        System.out.println("Can't find that ID!");
+    }
```

Comment:
hàm này có 2 tham số là id và newProduct và kiểu trả về là Product
và gán product.price = newProduct.price, ...
xóa System.out.println("Change completed") và System.out.println("Can't find that ID!");